pkg/chartutil/values_test.go

```diff
@@ -316,6 +322,8 @@ func TestCoalesceValues(t *testing.T) {
 		expect string
 	}{
 		{"{{.top}}", "yup"},
+		// The keys bottom, right, left, front, and back should not be present.
+		{"{{.side}}", ""},
```

Comment:
I saw that, but doesn't that signify the key is present but the value is empty? Is there currently a test method available that checks if a key is not present?